src/assets/pages/blogposts/post-44.html

```diff
@@ -0,0 +1,14 @@
+<p><em>Responda &agrave; Pesquisa de Usabilidade do Querido Di&aacute;rio e ajude a comunidade a implementar melhorias no front-end do projeto. O prazo para responder &eacute; at&eacute; dia 17 de outubro de 2023.&nbsp;</em></p>
+<p>A Open Knowledge Foundation Brasil (OKBR) est&aacute; com uma enquete aberta sobre a experi&ecirc;ncia das pessoas na utiliza&ccedil;&atilde;o da iniciativa do <a href="https://queridodiario.ok.org.br/">Querido Di&aacute;rio</a>. Para responder ao formul&aacute;rio &eacute; simples, basta clicar <a href="https://go.ok.org.br/pesquisa-qd">aqui</a>, leva apenas alguns minutos.&nbsp;</p>
```

Comment:
Pra gente garantir a acessibilidade, recomenda-se usar como texto do link uma palavra ou termo que sozinha deixe claro o contexto de para onde a pessoa será redirecionada em vez de "clique aqui".

O leitor de tela interpretaria como "Link, clique aqui", quando poderíamos usar outro texto, por exemplo "Link, formulário".

```suggestion
<p>A Open Knowledge Foundation Brasil (OKBR) est&aacute; com uma enquete aberta sobre a experi&ecirc;ncia das pessoas na utiliza&ccedil;&atilde;o da iniciativa do <a href="https://queridodiario.ok.org.br/">Querido Di&aacute;rio</a>. Responda ao <a href="https://go.ok.org.br/pesquisa-qd">formul&aacute;rio</a>, leva apenas alguns minutos.&nbsp;</p>
```